odohclient.go

```diff
@@ -13,22 +15,35 @@ import (
 	"github.com/miekg/dns"
 )
 
+const (
+	ODOH_CONFIG_PATH  = "/.well-known/odohconfigs"
+	DOH_CONTENT_TYPE  = "application/dns-message"
+	ODOH_CONTENT_TYPE = "application/oblivious-dns-message"
+)
+
 // ODoHClient is a Oblivious DNS client.
 type ODoHClient struct {
 	id         string
 	targetName string
 	targetPath string
+	targetPort string
 	proxy      *DoHClient
 
 	odohConfig       *odoh.ObliviousDoHConfig
+	odohConfigString string
 	odohConfigExpiry time.Time
 	mu               sync.Mutex
 }
 
 var _ Resolver = &DoHClient{}
 
-func NewODoHClient(id, endpoint, target string, opt DoHClientOptions) (*ODoHClient, error) {
-	proxy, err := NewDoHClient(id, endpoint, opt)
+func NewODoHClient(id, proxy, target, targetConfig string, opt DoHClientOptions) (*ODoHClient, error) {
+	if proxy == "" {
+		Log.Info("Attention! no ODoH proxy defined")
```

Comment:
```suggestion
		Log.Info("Attention! no ODoH proxy defined, using the target as proxy")
```

Should this be a `.Warn`?